go/vt/wrangler/migrater.go

```diff
@@ -184,11 +184,16 @@ func (wr *Wrangler) MigrateWrites(ctx context.Context, targetKeyspace, workflow
 	}
 	if !journalsExist {
 		mi.wr.Logger().Infof("No previous journals were found. Proceeding normally.")
-		sm, err := buildStreamMigrater(ctx, mi)
+		sm, err := buildStreamMigrater(ctx, mi, cancelMigrate)
 		if err != nil {
 			mi.wr.Logger().Errorf("buildStreamMigrater failed: %v", err)
 			return 0, err
 		}
+		if cancelMigrate {
+			mi.wr.Logger().Infof("Cancel was requested.")
+			mi.cancelMigration(ctx, sm)
```

Comment:
Error message in line 682 should be changed, it seems to be the same as the one above it.